--- Math-GSL-0.39/pod/Linalg.pod.orig	2016-10-01 08:24:23.000000000 +0200
+++ Math-GSL-0.39/pod/Linalg.pod	2017-07-02 21:48:43.871129403 +0200
@@ -16,7 +16,6 @@
                 gsl_linalg_hessenberg_unpack_accum
                 gsl_linalg_hessenberg_set_zero
                 gsl_linalg_hessenberg_submatrix
-                gsl_linalg_hessenberg
                 gsl_linalg_hesstri_decomp
                 gsl_linalg_SV_decomp
                 gsl_linalg_SV_decomp_mod
@@ -171,8 +170,6 @@
 
 =item gsl_linalg_hessenberg_submatrix
 
-=item gsl_linalg_hessenberg
-
 =item gsl_linalg_hesstri_decomp($A, $B, $U, $V, $work) - This function computes the Hessenberg-Triangular decomposition of the matrix pair ($A, $B). On output, H is stored in $A, and R is stored in $B. If $U and $V are provided (they may be null), the similarity transformations are stored in them. Additional workspace of length N is needed in the vector $work.
 
 =item gsl_linalg_SV_decomp($A, $V, $S, $work) - This function factorizes the M-by-N matrix $A into the singular value decomposition A = U S V^T for M >= N. On output the matrix $A is replaced by U. The diagonal elements of the singular value matrix S are stored in the vector $S. The singular values are non-negative and form a non-increasing sequence from S_1 to S_N. The matrix $V contains the elements of V in untransposed form. To form the product U S V^T it is necessary to take the transpose of V. A workspace of length N is required in vector $work. This routine uses the Golub-Reinsch SVD algorithm.
--- Math-GSL-0.39/pod/SF.pod.orig	2016-10-01 08:24:23.000000000 +0200
+++ Math-GSL-0.39/pod/SF.pod	2017-07-02 21:57:49.121123178 +0200
@@ -156,8 +156,6 @@
                gsl_sf_coupling_RacahW
                gsl_sf_coupling_9j_e
                gsl_sf_coupling_9j
-               gsl_sf_coupling_6j_INCORRECT_e
-               gsl_sf_coupling_6j_INCORRECT
                /;
 @EXPORT_dawson = qw/
                gsl_sf_dawson_e
--- Math-GSL-0.39/Build.PL.orig	2017-07-02 21:58:07.807789632 +0200
+++ Math-GSL-0.39/Build.PL	2017-07-02 22:12:26.074446499 +0200
@@ -56,7 +56,7 @@
 $ccflags .= try_cflags("-Wno-gnu");
 $ccflags .= try_cflags("-g");
 
-$swig_flags = "$gsl_conf->{gsl_cflags} $swig_flags";
+$swig_flags = "$gsl_conf->{gsl_cflags} $swig_flags -DGSL_DISABLE_DEPRECATED";
 
 if ( GSLBuilder::is_cygwin()  && $Config{shrpenv} =~ m{\Aenv LD_RUN_PATH=(.*)\Z} ) {
     $ldflags .= " -L$1 -lperl";
